smashed/mappers/tokenize.py

```diff
@@ -59,16 +61,37 @@ def __init__(
         if return_length:
             output_fields.append(f'{self.prefix}length')
 
+        if 'is_split_into_words' in tk_kwargs and return_word_ids:
```

Comment:
could we make this an explicit argument of the `__init__` method, and then add it to `tk_kwargs` in lines 30-32?

Reply:
done